src/stores/TierStore.js

```diff
@@ -86,6 +86,49 @@ class TierStore {
     }
   }
 
+  @action validateEditedTier = (property, index) => {
+    switch (property) {
+      case 'endTime':
+        let lessThanNextStart = true
+        const laterTime = validateLaterTime(this.tiers[index][property], this.tiers[index].startTime)
+
+        if (index < this.tiers.length - 1) {
+          lessThanNextStart = validateLaterOrEqualTime(this.tiers[index + 1].startTime, this.tiers[index][property])
+        }
+
+        this.validTiers[index][property] = lessThanNextStart && laterTime ? VALID : INVALID
+        return
+      case 'startTime':
+        let notLaterTime = true
+        const previousToEndTime = validateLaterTime(this.tiers[index].endTime, this.tiers[index][property])
+        const validTime = validateTime(this.tiers[index][property])
```

Comment:
Why does this check that the time is valid but the previous block doesn't?

Reply:
It could be checked, but is redundant as `endTime` is compared against `startTime`, so it's forced to be a _validTime_.

Anyway, if the idea is to check it for `endTime` too, then `validateTiers` action should be changed as well: https://github.com/poanetwork/ico-wizard/blob/72ffdc488dca905ea59f66ec76ae1625788ccac0/src/stores/TierStore.js#L82

Reply:
Nvm, if it was intended it's OK for me.